.github/workflows/tests.yml

```diff
@@ -178,13 +180,25 @@ jobs:
     - name: Install Ghidrathon
       run : |
         mkdir ./.github/ghidrathon
+        .github/ghidra/ghidra_10.3_PUBLIC/ghidraRun
+        mkdir ~/.ghidra/.ghidra_10.3_PUBLIC/Extensions
         wget "https://github.com/mandiant/Ghidrathon/archive/refs/tags/v2.1.0.zip" -O ./.github/ghidrathon/ghidrathon-2.1.0.zip
         unzip .github/ghidrathon/ghidrathon-2.1.0.zip -d .github/ghidrathon/
         workdir=$(pwd)
         /opt/gradle/gradle-7.3/bin/gradle -p ./.github/ghidrathon/Ghidrathon-2.1.0/ -PGHIDRA_INSTALL_DIR=$workdir/.github/ghidra/ghidra_10.3_PUBLIC 
-        unzip .github/ghidrathon/Ghidrathon-2.1.0/dist/*.zip -d $workdir/.github/ghidra/ghidra_10.3_PUBLIC/Extensions
+        unzip .github/ghidrathon/Ghidrathon-2.1.0/dist/*.zip -d ~/.ghidra/.ghidra_10.3_PUBLIC/Extensions
     - name: Install pyyaml
       run: sudo apt-get install -y libyaml-dev
     - name: Install capa
-      run: pip install -e .[dev]
+      run: | 
+        pip install -e .[dev]
+        git submodule update --init rules
+        python scripts/cache-ruleset.py rules/ cache/
+    - name: Get test data
+      run: | 
+        curl "https://raw.githubusercontent.com/mandiant/capa-testfiles/a37873c8a571b515f2baaf19bfcfaff5c7ef5342/Practical%20Malware%20Analysis%20Lab%2001-01.exe_" -o /tmp/pma_01-01.exe_      
```

Comment:
as above, the testfiles should be available from the submodule. there's nothing really wrong with this, though.

Reply:
see https://github.com/mandiant/capa/blob/33a3170bc4160dbfe925ef053e61823ece245996/.github/workflows/build.yml#L30-L58 for directly referencing test files from submodules. Let's not download the test file twice.

Reply:
Found that the `submodules: recursive` variable was initializing them. Changed the parameter to `true` and everything checked out.